"2019-05-08 11:42:46,654-INFO-case-日志信息:测试开始-[logger.py:32 ]"
"2019-05-08 11:42:46,655-ERROR-case-日志信息:测试报错-[logger.py:33 ]"
"2019-05-08 11:42:46,655-INFO-case-日志信息:测试结束-[logger.py:35 ]"
"2019-05-08 11:44:23,886-INFO-case-日志信息:测试开始-[logger.py:32 ]"
"2019-05-08 11:44:23,886-ERROR-case-日志信息:测试报错-[logger.py:33 ]"
"2019-05-08 11:44:23,887-DEBUG-case-日志信息:测试数据-[logger.py:34 ]"
"2019-05-08 11:44:23,887-INFO-case-日志信息:测试结束-[logger.py:35 ]"
"2019-05-08 11:53:47,125-INFO-case-日志信息:测试开始-[logger.py:32 ]"
"2019-05-08 11:53:47,125-ERROR-case-日志信息:测试报错-[logger.py:33 ]"
"2019-05-08 11:53:47,126-DEBUG-case-日志信息:测试数据-[logger.py:34 ]"
"2019-05-08 11:53:47,127-INFO-case-日志信息:测试结束-[logger.py:35 ]"
"2019-05-08 11:54:10,014-INFO-case-日志信息:测试开始-[logger.py:32 ]"
"2019-05-08 11:54:10,015-ERROR-case-日志信息:测试报错-[logger.py:33 ]"
"2019-05-08 11:54:10,015-DEBUG-case-日志信息:测试数据-[logger.py:34 ]"
"2019-05-08 11:54:10,015-INFO-case-日志信息:测试结束-[logger.py:35 ]"
"2019-05-08 11:54:10,237-INFO-__main__-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 11:54:10,237-INFO-__main__-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 11:54:10,484-INFO-__main__-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 11:54:10,484-INFO-__main__-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 11:54:11,377-INFO-__main__-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 11:54:11,378-INFO-__main__-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 11:54:11,496-INFO-__main__-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 11:54:11,496-INFO-__main__-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 11:54:11,603-INFO-__main__-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 11:54:11,604-INFO-__main__-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 11:54:42,498-INFO-__main__-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 11:54:42,498-INFO-__main__-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 11:54:42,625-INFO-__main__-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 11:54:42,625-INFO-__main__-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 11:54:42,746-INFO-__main__-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 11:54:42,747-INFO-__main__-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 11:54:42,852-INFO-__main__-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 11:54:42,853-INFO-__main__-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 11:54:42,951-INFO-__main__-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 11:54:42,951-INFO-__main__-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 11:57:57,954-INFO-__main__-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 11:57:57,955-INFO-__main__-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 11:57:58,112-ERROR-__main__-日志信息:报错：'{"status":1,"code":"10001","data":null,"msg":"登录成功"}' != {'Server': 'nginx/1.12.0', 'Date': 'Wed, [299 chars]GMT'}-[test_login.py:33 ]"
"2019-05-08 11:57:58,113-INFO-__main__-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 11:57:58,593-ERROR-__main__-日志信息:报错：'{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"}' != {'Server': 'nginx/1.12.0', 'Date': 'Wed, [131 chars]ive'}-[test_login.py:33 ]"
"2019-05-08 11:57:58,594-INFO-__main__-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 11:57:58,691-ERROR-__main__-日志信息:报错：'{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}' != {'Server': 'nginx/1.12.0', 'Date': 'Wed, [236 chars]GMT'}-[test_login.py:33 ]"
"2019-05-08 11:57:58,692-INFO-__main__-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 11:57:58,792-ERROR-__main__-日志信息:报错：'{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"}' != {'Server': 'nginx/1.12.0', 'Date': 'Wed, [236 chars]GMT'}-[test_login.py:33 ]"
"2019-05-08 11:57:58,793-INFO-__main__-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 11:58:47,061-INFO-__main__-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 11:58:47,062-INFO-__main__-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 11:58:47,191-INFO-__main__-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 11:58:47,192-INFO-__main__-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 11:58:47,314-INFO-__main__-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 11:58:47,315-INFO-__main__-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 11:58:47,413-INFO-__main__-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 11:58:47,413-INFO-__main__-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 11:58:47,515-INFO-__main__-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 11:58:47,516-INFO-__main__-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 12:40:00,919-INFO-__main__-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 12:40:00,921-INFO-__main__-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 12:40:00,922-DEBUG-API_0419.common.http_request-日志信息:请求url：http://test.lemonban.com/futureloan/mvc/api/member/login-[http_request.py:40 ]"
"2019-05-08 12:40:00,922-DEBUG-API_0419.common.http_request-日志信息:请求data：{'mobilephone': '13662278421', 'pwd': '123456789'}-[http_request.py:41 ]"
"2019-05-08 12:40:01,079-INFO-__main__-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 12:40:01,079-INFO-__main__-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 12:40:01,080-DEBUG-API_0419.common.http_request-日志信息:请求url：http://test.lemonban.com/futureloan/mvc/api/member/login-[http_request.py:40 ]"
"2019-05-08 12:40:01,080-DEBUG-API_0419.common.http_request-日志信息:请求data：{}-[http_request.py:41 ]"
"2019-05-08 12:40:01,868-INFO-__main__-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 12:40:01,868-INFO-__main__-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 12:40:01,869-DEBUG-API_0419.common.http_request-日志信息:请求url：http://test.lemonban.com/futureloan/mvc/api/member/login-[http_request.py:40 ]"
"2019-05-08 12:40:01,869-DEBUG-API_0419.common.http_request-日志信息:请求data：{'mobilephone': '1355112375', 'pwd': '123456789'}-[http_request.py:41 ]"
"2019-05-08 12:40:01,965-INFO-__main__-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 12:40:01,965-INFO-__main__-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 12:40:01,966-DEBUG-API_0419.common.http_request-日志信息:请求url：http://test.lemonban.com/futureloan/mvc/api/member/login-[http_request.py:40 ]"
"2019-05-08 12:40:01,966-DEBUG-API_0419.common.http_request-日志信息:请求data：{'mobilephone': '13662278421', 'pwd': '12345678'}-[http_request.py:41 ]"
"2019-05-08 12:40:02,072-INFO-__main__-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 12:40:02,072-INFO-__main__-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 13:02:25,561-INFO-__main__-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 13:02:25,562-INFO-__main__-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 13:02:25,735-INFO-__main__-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 13:02:25,736-INFO-__main__-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 13:02:26,702-INFO-__main__-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 13:02:26,702-INFO-__main__-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 13:02:26,795-INFO-__main__-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 13:02:26,795-INFO-__main__-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 13:02:26,888-INFO-__main__-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 13:02:26,889-INFO-__main__-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 13:15:44,290-INFO-API_0419.test_cases.test_login-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 13:15:44,291-INFO-API_0419.test_cases.test_login-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 13:15:44,475-INFO-API_0419.test_cases.test_login-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 13:15:44,475-INFO-API_0419.test_cases.test_login-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 13:15:44,602-INFO-API_0419.test_cases.test_login-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 13:15:44,603-INFO-API_0419.test_cases.test_login-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 13:15:44,715-INFO-API_0419.test_cases.test_login-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 13:15:44,715-INFO-API_0419.test_cases.test_login-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 13:15:44,820-INFO-API_0419.test_cases.test_login-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 13:15:44,820-INFO-API_0419.test_cases.test_login-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 13:28:49,259-INFO-test_login-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 13:28:49,259-INFO-test_login-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 13:28:49,374-INFO-test_login-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 13:28:49,374-INFO-test_login-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 13:28:49,473-INFO-test_login-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 13:28:49,474-INFO-test_login-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 13:28:49,594-INFO-test_login-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 13:28:49,595-INFO-test_login-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 13:28:49,700-INFO-test_login-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 13:28:49,701-INFO-test_login-日志信息:测试后置-[test_login.py:38 ]"
"2019-05-08 13:34:34,420-INFO-test_login-日志信息:测试前置-[test_login.py:19 ]"
"2019-05-08 13:34:34,420-INFO-test_login-日志信息:开始测试：正常登录-[test_login.py:24 ]"
"2019-05-08 13:34:34,544-INFO-test_login-日志信息:结束测试：正常登录-[test_login.py:35 ]"
"2019-05-08 13:34:34,544-INFO-test_login-日志信息:开始测试：参数为空-[test_login.py:24 ]"
"2019-05-08 13:34:34,661-INFO-test_login-日志信息:结束测试：参数为空-[test_login.py:35 ]"
"2019-05-08 13:34:34,661-INFO-test_login-日志信息:开始测试：用户名错误-[test_login.py:24 ]"
"2019-05-08 13:34:34,767-INFO-test_login-日志信息:结束测试：用户名错误-[test_login.py:35 ]"
"2019-05-08 13:34:34,768-INFO-test_login-日志信息:开始测试：密码错误-[test_login.py:24 ]"
"2019-05-08 13:34:34,888-INFO-test_login-日志信息:结束测试：密码错误-[test_login.py:35 ]"
"2019-05-08 13:34:34,889-INFO-test_login-日志信息:测试后置-[test_login.py:38 ]"
